src/_routing/_components/RootStackNavigator.tsx

```diff
@@ -206,6 +207,14 @@ export const RootStackNavigator = () => {
               title: i18n.t('ONBOARDING.FAMILY.ADD_MEMBER.TITLE'),
             }}
           />
+          <RootStack.Screen
```

Comment:
According to the designs, we should also show the bottom navigation.

Reply:
decided not to do this yet, but do this as one refactor (discussed with Helena)